lib/Chat/ChatManager.php

```diff
@@ -751,6 +755,129 @@ public function editMessage(Room $chat, IComment $comment, Participant $particip
 		);
 	}
 
+	public function pinMessage(Room $chat, IComment $comment, Participant $participant, int $pinUntil): ?IComment {
```

Comment:
```suggestion
	public function pinMessage(Room $chat, IComment $comment, Participant $participant, int $pinUntil = 0): ?IComment {
```

Default to 0 then you can avoid the if below